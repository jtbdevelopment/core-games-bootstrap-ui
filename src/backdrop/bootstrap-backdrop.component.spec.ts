import {TestBed} from '@angular/core/testing';
import {JTBModalBackdrop} from './bootstrap-backdrop.component';


describe('Component:  bootstrap backdrop component', () => {
  let fixture;
  let modal;
  beforeEach(() => {
    TestBed.configureTestingModule({
      declarations: [
        JTBModalBackdrop,
      ],
    });
    TestBed.compileComponents();
    fixture = TestBed.createComponent(JTBModalBackdrop);
    fixture.detectChanges();
  });

  it('displays backdrop', () => {
    expect(fixture).toMatchSnapshot();
  });
});
