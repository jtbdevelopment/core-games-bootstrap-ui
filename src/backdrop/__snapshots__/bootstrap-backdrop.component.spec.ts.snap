// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component:  bootstrap backdrop component displays backdrop 1`] = `
<jtb-modal-backdrop
  backdropClass={[Function String]}
>
  
</jtb-modal-backdrop>
`;
